django_pgschemas/schema.py

```diff
@@ -1,19 +1,20 @@
-from contextlib import contextmanager
-from contextvars import ContextVar
-from typing import Any, Iterator
-
-from asgiref.sync import sync_to_async
+from contextlib import ContextDecorator, contextmanager
+from contextvars import ContextVar, Token
+from functools import lru_cache
+from typing import Any, Iterator, Literal
 
 from django_pgschemas.routing.info import RoutingInfo
 from django_pgschemas.signals import schema_activate
 
 
-class Schema:
+class Schema(ContextDecorator):
```

Comment:
Making it a context decorator causes it to be an empty string in the templates,
![image](https://github.com/user-attachments/assets/225a6708-2c2f-4e8e-a2ef-3201074528be)